types/ngx.leaflet.offline/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-object-literal-type-assertion": false,
```

Comment:
If these are required, can you please suppress them on the lines where they occur.

Reply:
ok

Reply:
if they are necessary.

Reply:
Can you please remove these entries?